modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,66 @@
+
+[NOTE]
+====
+If network policies are configured incorrectly they can disrupt mesh traffic. When performing the migration, exercise caution when creating network policies to prevent traffic disruption. See "Network Policy migration" for more information.
+====
+
+== Control planes and sidecar injection
+
+During the migration, only one control plane must perform sidecar injection. This is achieved by correctly using injection labels. See "Installing the Sidecar" for more information.
+
+[NOTE]
+====
+During the migration, you must disable the 2.6 injector. Use the `maistra.io/ignore-namespace: "true"` label to prevent the 2.6 control plane from injecting a proxy in the namespace.
+====
+
+If both control planes try to perform sidecar injection the proxy will not start and the migration cannot be completed.
+
+== Control planes and label selection
+
+For {SMProduct} 3.0, you must decide if you want to use the `istio.io/rev` or `istio-injection` label to configure sidecar injection. See "Installing the Sidecar" for more information about how to use the labels.
+
+In the {SMProduct} 2.6 installation, the way member selection was configured in the `ServiceMeshMemberRoll` resource can impact how injection labels are used in the {SMProduct} 3.0 installation.
+
+By default, in a 2.6 installation, the `spec.memberSelectors` field in the `ServiceMeshMemberRoll` resource is configured to match the `istio-injection=enabled` label and all of the data plane namespaces in a 2.6 installation have the `istio-injection=enabled` label applied. If you are using the default 2.6 installation settings, you can either keep using that label or switch to the `istio.io/rev` label for the 3.0 installation.
+
+If the `spec.memberSelectors` field in the `ServiceMeshMemberRoll` resource is not configured to match the `istio-injection=enabled` label and the 2.6 data plane namespace uses a custom label you must add the `istio.io/rev` or `istio-injection` label during the migration. The custom labels defined in the `spec.memberSelectors` parameter of the `ServiceMeshMemberRoll` resource have no effect on sidecar injection in the {SMProduct} 3 installation and cannot be used. 
+
+In the 2.6 installation, if projects were added to the mesh by manually creating the `ServiceMeshMember` resource you must add the `istio.io/rev` or `istio-injection` label to the project namespaces during the migration.
+
+== Cluster-Wide migration methods
+
+Migrating from {SMProductName} {SMv2Version} to {SMProduct} 3.0 can be performed using one of the following methods:
+
+* Applying the `istio.io/rev` label
+* Applying the `istio-injection=enabled` label
+* Performing a simple migration using the `istio-injection=enabled` label; however, the simple migration method should not be used in a production environment because there will be traffic disruptions.
```

Comment:
nit: I don't think it's guaranteed there will be traffic disruptions so maybe we can soften the language here.
```suggestion
* Performing a simple migration using the `istio-injection=enabled` label; however, the simple migration method should not be used in a production environment because there could be traffic disruptions.
```

Reply:
done